test/bulleted_list_2_space_start_4_space_indent.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Please use '-' instead of '\_' for file names. The ones in the repo that use '_' are legacy.